packages/angular_devkit/build_angular/src/builders/browser-esbuild/options.ts

```diff
@@ -20,6 +20,20 @@ import { Schema as BrowserBuilderOptions, OutputHashing } from './schema';
 
 export type NormalizedBrowserOptions = Awaited<ReturnType<typeof normalizeOptions>>;
 
+/** Internal infrastructure settings hidden from builder schema but available when invoked programmatically. */
+export interface InfrastructureSettings {
+  /** Whether or not to write output files to disk. */
+  write?: boolean;
+
+  /**
+   * Entry points to use for the compilation. Incompatible with `main`, which must not be provided. May be relative or absolute paths.
+   * If given a relative path, it is resolved relative to the current workspace and will generate an output at the same relative location
+   * in the output directory. If given an absolute path, the output will be generated in the root of the output directory with the same base
+   * name.
+   */
+  entryPoints?: Set<string>;
```

Comment:
I am okay with this for now as a workaround, but I am not a big fan of additional non infra related option in the infra settings.

Reply:
Moved to `InternalOptions` distinct from `InfrastructureSettings`.